int32   id
float64 range
float32 range_lat
float64 range_lon
float64 angle
float64 vel
float64 v_lat
float64 v_lon
int32   status
float64 w
float64 l
int32   devid